<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>29</y>
      <width>571</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="equationWidget" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addEquationButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Equation1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Equation2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Equation3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>110</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>170</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
